### ISSUE TITLE
Windows 10 IoT start but then black screen

### ISSUE THREAD

**Issue by Takaao1** [ms-iot/iot-adk-addonkit#321]
I have installed the application from the factory where i work, on the image .ffu

Then, i have installed the windows PE on the hard drive, and added the .ffu image on the root of it.

At least, i have put the hard drive on the computer, who have launched a cmd console, i have done this command for launch the .ffu file :

dism.exe /apply-image /ImageFile:Flash.ffu /ApplyDrive:\\.\PhysicalDrive0 /skipplatformcheck
Then, i have turn off the computer, and put out the hard drive

I have launched the computer, a Windows 10 IoT page came like this :
![image](https://user-images.githubusercontent.com/43552973/58874759-e761b700-86c9-11e9-8f53-671c76927ba6.png)

And after, i have a black screen.

I'm turning off / on everytime the computer, but i have still the black screen.

Why ? What Am I supposed to do ???

I'm so confused, i can do nothing and i'm at 99 % of my project now ...

I need your help as quick as possible, because the app from the factory has to work with this Windows 10 IoT Core!

**Comment by parameshbabu** [comment]
Hi
App install instructions are here.
https://docs.microsoft.com/en-us/windows/iot-core/develop-your-app/appinstaller

Is your app set as default?

Regards,
Paramesh
________________________________

**Comment by Takaao1** [comment]
I guess yes, i have followed all the instructions of it , so it's why i'm confused because the installation on the computer is not working (still have this black screen).
What am I supposed to do ?

**Comment by Takaao1** [comment]
Just , the package application was on .appxbundle, is it a problem ? Is it the cause of the problem ? I was thinking about it

**Comment by parameshbabu** [comment]
Appxbundle is not an issue ( except that it bloats the package size for unnecessary binaries for other arch).
Have you tried installing the appx independently and ensure that it works on the device?
Are you building a retail image or a test image?

**Comment by Takaao1** [comment]
Yes i tried to install it independatly, but it doesn t work, I have the same problem ( black screen )

Obtenez Outlook pour iOS<https://aka.ms/o0ukef>
________________________________
De : Paramesh Babu <notifications@github.com>
Envoyé : Wednesday, June 5, 2019 5:41:43 AM
À : ms-iot/iot-adk-addonkit
Cc : Takaao1; Comment
Objet : Re: [ms-iot/iot-adk-addonkit] Windows 10 IoT start but then black screen (#321)

Appxbundle is not an issue ( except that it bloats the package size for unnecessary binaries for other arch).
Have you tried installing the appx independently and ensure that it works on the device?
Are you building a retail image or a test image?

**Comment by parameshbabu** [comment]
I think the application needs to be debugged on why it’s not launched. This is not an imaging issue. Recommend to manually deploy the app and debug.

Regards,
Paramesh
________________________________

**Comment by Takaao1** [comment]
By the Way it s a test image and not a retail image, and how am i supposed to do it ? Because the application was made by a developer from the factory and she works with other devices, so are you sure ?

Obtenez Outlook pour iOS<https://aka.ms/o0ukef>
________________________________
De : Paramesh Babu <notifications@github.com>
Envoyé : Wednesday, June 5, 2019 8:46:41 AM
À : ms-iot/iot-adk-addonkit
Cc : Takaao1; Comment
Objet : Re: [ms-iot/iot-adk-addonkit] Windows 10 IoT start but then black screen (#321)

I think the application needs to be debugged on why it’s not launched. This is not an imaging issue. Recommend to manually deploy the app and debug.

Regards,
Paramesh
________________________________

**Comment by Takaao1** [comment]
And what do you mean by " Recommend to manually deploy the app and debug " ?? 
Now i'm lost.
I don't know exactly what steps i'm supposed to do

**Comment by parameshbabu** [comment]
https://docs.microsoft.com/en-us/windows/iot-core/develop-your-app/remotedebugging should help you with your debugging process.

**Comment by Takaao1** [comment]
How do you want from me to do that, I can't because i Have a black screen, and the only way for me to have my IoT Core good is to don't put the app on it , so it will be usless to do that without app
So for me, it won't work, right?

**Comment by parameshbabu** [comment]
Build the device without your app and boot it and then manually install your app ( see https://docs.microsoft.com/en-us/windows/iot-core/develop-your-app/appinstaller ) and launch to see why your app is not running. You are debug at this time as well. Once you have got your app running on the device then you can create the image including that app.

**Comment by Takaao1** [comment]
I built the device without the app, but now when i'm at the home page of Windows 10 IoT Core, i need to access to Wi-Fi (But it found nothing, why ?) And when i try to access to the Windows Device Portal, it doesn't work too
With the username : administrator
and the password : p@ssw0rd

How am I supposed to do to works ?

**Comment by IsmailHassani** [comment]
Check this [https://social.msdn.microsoft.com/Forums/en-US/5e69a7eb-b8f6-4ac8-a150-025e7d045842/uwp-application-is-not-installed?forum=WindowsIoT](https://social.msdn.microsoft.com/Forums/en-US/5e69a7eb-b8f6-4ac8-a150-025e7d045842/uwp-application-is-not-installed?forum=WindowsIoT) where i had the same issue. Maybe it helps
